### ISSUE TITLE
Link with tag <a> is not visible 

### ISSUE THREAD

**Issue by ferdavs** [PrivacyApps/html-textview#129]
but I can click it.

**Comment by GLodi** [comment]
It may have happened to me the same thing.
I'm building a GitHub app that has a light theme and a dark theme. If the user is using the light theme, then all links are displayed in the regular blue color, but with the dark theme all links are hidden. I can still click on them though. 
I've still no idea why this happens. I've fixed it by changing the text's html with Jsoup (in Kotlin): 

`
val doc: Document = Jsoup.parse(comment.bodyHtml)

val links: Elements = doc.select("a")
for (link in links) {
    val e: Element = Jsoup.parse("<font color='#326fba'>$link</font>")
    link.replaceWith(e)
}
row_comment_comment.setHtml(doc.html(), HtmlHttpImageGetter(row_comment_comment))
`

Basically it puts all <a> tags within a <font> tag with a specific color. I've also tried to add a style attribute to the <a> tag, but it didn't work.
